packages/kbn-pm/dist/index.js

```diff
@@ -6639,7 +6639,17 @@ class ToolingLogTextWriter {
     }
 
     if (this.ignoreSources && msg.source && this.ignoreSources.includes(msg.source)) {
-      return false;
+      if (msg.type === 'write') {
+        const txt = (0, _util.format)(msg.args[0], ...msg.args.slice(1)); // Ensure that Elasticsearch deprecation log messages from Kibana aren't ignored
+
+        if (/\[deprecation\]\[elasticsearch\]/.test(txt)) {
+          return true;
+        } else {
+          return false;
+        }
```

Comment:
Looks like this might be the older built version of kbn-pm before you made modifications